codemp/game/w_saber.c

```diff
@@ -109,6 +109,16 @@ qboolean G_CanBeEnemy( gentity_t *self, gentity_t *enemy )
 	if ( !self->inuse || !enemy->inuse || !self->client || !enemy->client )
 		return qfalse;
 
+	if (self->client->ps.duelInProgress && self->client->ps.duelIndex != enemy->s.number)
+	{ //dueling but not with this person
+		return qfalse;
+	}
+
+	if (enemy->client->ps.duelInProgress && enemy->client->ps.duelIndex != self->s.number)
+	{ //other guy dueling but not with me
+		return qfalse;
+	}
```

Comment:
This was removed from jasdk_modbase before its git history. We don't know why, so let's hope bringing it back is the right move.